rotkehlchen/chain/evm/constants.py

```diff
@@ -6,3 +6,20 @@
 ZERO_ADDRESS = string_to_evm_address('0x0000000000000000000000000000000000000000')
 ETH_SPECIAL_ADDRESS = string_to_evm_address('0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE')
 GENESIS_HASH = deserialize_evm_tx_hash('0x' + '0' * 64)  # hash for transactions in genesis block
+
+EMPTY_RECEIPT = {
+    'blockHash': '0x' + '0' * 64,
```

Comment:
can you maybe store this in a variable and use it here, in genesis hash and below?